config.js

```diff
@@ -97,6 +97,8 @@ config.STORE_PERF_TEST_INTERVAL = 60 * 60 * 1000; // perform test_store_perf eve
 config.CLOUD_MAX_ALLOWED_IO_TEST_ERRORS = 3;
 
 
+config.DISABLE_DEV_RANDOM_SEED = 'false';
```

Comment:
@naveenpaul1 @guymguym 
Guy suggested to convert the config to - 
config.ENABLE_DEV_RANDOM_SEED = 'false'; 
because we always (on containerized and non containerized) won't enable it.
@naveenpaul1 WDYT? this will make the code on manage_nsfs.js and health.js redundant,
and we should adjust the nb_native.js code